modules/account-lib/src/coin/algo/utils.ts

```diff
@@ -231,19 +232,19 @@ export class Utils implements BaseUtils {
    * @param seed 32 bytes long seed
    * @returns { sk, pk } object with secretKey and publicKey
    */
-  keyPairFromSeed(seed: Uint8Array) {
+  keyPairFromSeed(seed: Uint8Array): any {
```

Comment:
Why the return is `any` and not a strong typed object as in `secretKeyToMnemonic` and `seedFromMnemonic`